src/web/core/src/main/java/org/geoserver/web/security/AccessDataRuleListView.java

```diff
@@ -0,0 +1,108 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.web.security;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.wicket.ajax.AjaxRequestTarget;
+import org.apache.wicket.ajax.form.AjaxFormComponentUpdatingBehavior;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.form.CheckBox;
+import org.apache.wicket.markup.html.list.ListItem;
+import org.apache.wicket.markup.html.list.PageableListView;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.PropertyModel;
+
+public class AccessDataRuleListView extends PageableListView<DataAccessRuleInfo> {
+
+    private List<CheckBox> checkBoxes;
+    private Boolean selectAll;
+    private boolean isWs;
+
+    public AccessDataRuleListView(String id, IModel<List<DataAccessRuleInfo>> model, boolean isWs) {
+        super(id, model, 15);
```

Comment:
15 per page is inconsistent with the pageable tables in GeoServer, see GeoServerTablePanel, using 25 by default.
I suggest using the same.

Also, what happens if someone pages around? Are the edits retained?
Finally, UI wise there is an inconsistency, here is what most tables look like:

![Selezione_988](https://user-images.githubusercontent.com/325433/68600803-6cd06b00-04a3-11ea-9a40-b5bfd2a6d2fa.png)

while the access panel looks like this:

![Selezione_989](https://user-images.githubusercontent.com/325433/68600817-73f77900-04a3-11ea-8f5c-4ae309d0e2a8.png)

I understand that using GeoServerTablePanel for an editable, pageable view is hard, but at least they should look the same.

Also... does it really need to page? The roles are presented in random order, so if there are 1000 roles one :
- Would have a hard time finding a specific one (at lest order them alphabetically)
- Would have a hard time locating the active rules

Wondering if this one should just be made non pageable, under the assumption that roles will not be too many. @nmco opinions?

Reply:
removed pagination